test/test-classTetrisMoves.js

```diff
@@ -44,6 +44,24 @@ describe("All moves tetromino", function () {
             -------
         `)
         assert.deepEqual(elementTetris.tetromino.coordinates, expectedMoveLeft)
+        elementTetris.tick()
+        let expectedTick = tetris.getCoordinates(`
+            -------
+            -------
+            --XX---
+            ---X---
+            ---X---
+        `)
+        assert.deepEqual(elementTetris.tetromino.coordinates, expectedTick)
+        elementTetris.tick()
```

Comment:
you should also check the field before and after this tick

Reply:
I do a status check before tick
string 39

Reply:
you do not check the field currently
check the field